.dev/format.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import subprocess
+import argparse
+
+WHITELIST = [
```

Comment:
Can we put these paths into a separate file (e.g. `.dev/paths_to_format.txt`) and then read that file both in format.py and format.sh? To avoid duplication

Reply:
I was actually thinking to remove the format.sh after a while, seeing that the python script performs as intended. So I could use the same list/file for both manual/ci formatting and for pre-commit.
We can do the split if you prefer to keep the format.sh and perhaps AI can help with the shell programming 😄

Reply:
I think I would prefer to keep format.sh for now. The shell command `cat` is probably sufficient to print the content of the file, so maybe `local whitelist="${cat ${PCL_DIR}/.dev/paths_to_format.txt}"`?